File: odd_square.js
#!/usr/bin/env node
// square w/o expt or multiplication

function oddSquare(n) {
  var num = n;
  for (var i = 1; i < n; i++) {
    num += n;
  }
  return num;
}

console.log(oddSquare(4));
console.log(oddSquare(3));
console.log(oddSquare(9));

File: odd_square.py
#!/usr/bin/env python
# http://programmingpraxis.com/2013/02/26/an-odd-way-to-square/

def square(n):
  result = n
  for _ in xrange(1,n):
    result += n
  return result

print(square(3))
print(square(9))
print(square(5))
